### DISCUSSION TITLE
WinXP copy dialog

### DISCUSSION THREAD

**Discussion by Anixx** [valinet/ExplorerPatcher#674] [Ideas]
This may be of interest.
https://winclassic.boards.net/thread/1057/interesting-discovery-windows-progress-dialog

**Comment by valinet** [comment]
Yeah, interesting, but what's the actual "discovery"? The post is kind of useless... has the OP found a way to enable it, or just made the observation (that everybody knows) that the XP-era copy dialog shows when installing a driver manually...?
